monkey/infection_monkey/post_breach/clear_command_history/linux_clear_command_history.py

```diff
@@ -1,9 +1,15 @@
+import logging
 import subprocess
+from typing import Iterable, Union
 
+from common.common_consts.post_breach_consts import POST_BREACH_CLEAR_CMD_HISTORY
+from common.common_consts.timeouts import LONG_REQUEST_TIMEOUT
 from infection_monkey.utils.environment import is_windows_os
 
+logger = logging.getLogger(__name__)
 
-def get_linux_commands_to_clear_command_history():
+
+def get_linux_commands_to_clear_command_history() -> Union[Iterable[str], str]:
     if is_windows_os():
         return ""
```

Comment:
Can we make this return `[]`? Then we can always return `Iterable[str]`. I'm not sure if that will introduce any bugs.